src/test/java/com/vanniktech/dependency/graph/generator/DependencyGraphGeneratorPluginTest.kt

```diff
@@ -213,7 +216,7 @@ class DependencyGraphGeneratorPluginTest {
         .withPluginClasspath()
         .withGradleVersion("5.0")
         .withProjectDir(testProjectDir.root)
-        .withArguments("generateDependencyGraph", "generateProjectDependencyGraph")
+        .withArguments("generateDependencyGraph", "generateProjectDependencyGraph", "app:generateProjectDependencyGraph")
```

Comment:
This will trigger the task to generate project dependency graph on sub project `app`.